### PR TITLE
Update Selenium Manager (beta) page

### PR DESCRIPTION
**Pull request by bonigarcia**
### Description
This  PR updates the English version of the Selenium Manager (beta) page.

Although some parts of this documentation are a bit lengthy, I recommend explaining it in detail to understand better Selenium Manager and its relationship with the rest of Selenium components. Also, specific examples made this page a bit longer, but I believe they help understand the presented concepts (e.g., driver management, caching, etc.).

### Motivation and Context
This PR implements https://github.com/SeleniumHQ/selenium/issues/11695 of the Selenium Manager project.

### Types of changes

- [X] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [X] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [ ] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

**Comment by iamalittletester** [comment]
It looks good to me generally but i would change a few things to better highlight some aspects:
- in the paragraph right after the title i would specify that this manages now comes out of the box, highlighting that the user doesn't need to do anything extra to get the manager (except to download Selenium obviously)
- in the Motivation section i would change the tense to past, to suggest that those actions (like downloading the driver, setting the path) are actions that users previously needed to do. Currently, when using the latest Selenium versions, the users don't need to do all those things. Also, in the last phrase of this section i would again emphasize that the manager comes out of the box, with the Selenium library. 
- the above 2 lines are based on wanting to quickly make the users understand the basics and important aspects of the manager, rather than provide this information after many paragraphs
- in the Usage section, where you specify that the users can continue to manage their own drivers, or have this done automatically, i would inverse the order of these 2: we want to encourage the users to use the 'automated' mode, so we should put that as the first option in this paragraph
- in the Configuration section, i would start with when would a user need to do the setup mentioned in this section. Or maybe i understand it wrong, maybe it actually describes how the manager works under the hood. This is confusing i think, so maybe describe why the user needs to do what is described there or whether that's the inner workings of the manager

File: website_and_docs/content/documentation/selenium_manager.en.md [hunk 1]
```diff
@@ -3,42 +3,52 @@ title: "Selenium Manager (Beta)"
 linkTitle: "Selenium Manager"
 weight: 11
 description: >
-    Selenium Manager is a binary tool implemented in Rust that provides automated driver and browser management for Selenium.
+    Selenium Manager is a binary tool implemented in Rust that provides automated driver and browser management for Selenium. Selenium bindings use this tool by default, so you do not need to download it or add anything to your code or do anything else to use it.
```
Comment (titusfortner) [inline website_and_docs/content/documentation/selenium_manager.en.md:6]:
```suggestion
    Selenium Manager is a command-line tool implemented in Rust that provides automated driver and browser management for Selenium. Selenium bindings use this tool by default, so you do not need to download it or add anything to your code or do anything else to use it.
```

File: website_and_docs/content/documentation/selenium_manager.en.md [hunk 2]
```diff
@@ -3,42 +3,52 @@ title: "Selenium Manager (Beta)"
...
+Selenium uses the native support implemented by each browser to carry out the automation process. For this reason, Selenium users need to place a component called _driver_ (e.g., chromedriver, geckodriver, msedgedriver, etc.) between the script using the Selenium API and the browser. For many years, managing these drivers was a manual process for Selenium users. This way, they had to download the required driver for a browser (e.g., chromedriver for Chrome, geckodriver for Firefox, etc.) and place it in the `PATH` or export the driver path as a system property (e.g., Java, JavaScript, etc.). But this process was cumbersome and led to maintainability issues.
```
Comment (titusfortner) [inline website_and_docs/content/documentation/selenium_manager.en.md]: nit: use either "e.g.," or "etc." — the statement stops making sense when using both.

File: website_and_docs/content/documentation/selenium_manager.en.md [hunk 3]
```diff
@@ -3,42 +3,52 @@ title: "Selenium Manager (Beta)"
...
+    Selenium Manager is a binary tool implemented in Rust that provides automated driver and browser management for Selenium. Selenium bindings use this tool by default, so you do not need to download it or add anything to your code or do anything else to use it.
 ---
 
 ## Motivation
-Selenium uses the native support implemented by each browser to carry out the automation process. For this reason, Selenium users need to place a component called _driver_ (e.g., chromedriver, geckodriver, msedgedriver, etc.) between the script using the Selenium API and the browser. For many years, managing these drivers was a manual process for Selenium users. This way, they must download the required driver for a browser (e.g., chromedriver for Chrome, geckodriver for Firefox, etc.) and place it in the `PATH` or export the driver path as a system property (e.g., Java, JavaScript, etc.). But this process is cumbersome and leads to maintainability issues.
+**TL;DR**: **Selenium Manager** is the official *driver manager* of the Selenium project, and it is shipped out of the box with every Selenium release as of version 4.6.
```
Comment (titusfortner) [inline website_and_docs/content/documentation/selenium_manager.en.md]:
```suggestion
**TL;DR**: **Selenium Manager** is the official *driver manager* of the Selenium project, and it is shipped out of the box with every Selenium release as of version 4.6. Some features are unreliable before Selenium 4.11, though, so please use the latest version of Selenium to get the most use out of this feature.
```

File: website_and_docs/content/documentation/selenium_manager.en.md [hunk 4]
```diff
@@ -3,42 +3,52 @@ title: "Selenium Manager (Beta)"
...
 
 ## Usage
-Selenium Manager is a CLI (command line interface) tool implemented in Rust to allow cross-platform execution and compiled for Windows, Linux, and macOS. The Selenium Manager binaries are shipped with each Selenium release. This way, each Selenium binding language invokes Selenium Manager to carry out the automated driver and browser management explained in the following sections. 
+**TL;DR**: Selenium Manager is used by the Selenium bindings when the drivers (chromedriver, geckodriver, etc.) are unavailable.
+
+Driver management through Selenium Manager is *opt-in* for the Selenium bindings. Thus, users can continue managing their drivers manually (putting the driver in the `PATH` or using system properties) or rely on a third-party *driver manager* to do it automatically. Selenium Manager only operates as a fallback: if no driver is provided, Selenium Manager will come to the rescue.
```
Comment (titusfortner) [inline website_and_docs/content/documentation/selenium_manager.en.md:22]:
```suggestion
Driver management through Selenium Manager is currently *opt-in* for the Selenium bindings. Thus, users can continue managing their drivers manually (putting the driver in the `PATH` or using system properties) or rely on a third-party *driver manager* to do it automatically. Selenium Manager only operates as a fallback: if no driver is provided, Selenium Manager will come to the rescue.
```

**Comment by bonigarcia** [comment]
Thanks a lot, @iamalittletester, for your feedback. I have just committed a second patch to this PR reviewing the content according to your comments. Also, I have included several **TL;DR** paragraphs at the beginning of the main sections to highlight the main ideas.

File: website_and_docs/content/documentation/selenium_manager.en.md [hunk 5]
```diff
@@ -3,42 +3,52 @@ title: "Selenium Manager (Beta)"
...
 
-Driver management through Selenium Manager is *opt-in* for the Selenium bindings. Thus, users can continue managing their drivers manually (putting the driver in the `PATH` or using system properties) or rely on a third-party *manager* to do it automatically. Selenium Manager only operates as a fallback: if no driver is provided, Selenium Manager will come to the rescue.
+Selenium Manager is a CLI (command line interface) tool implemented in Rust to allow cross-platform execution and compiled for Windows, Linux, and macOS. The Selenium Manager binaries are shipped with each Selenium release. This way, each Selenium binding language invokes Selenium Manager to carry out the automated driver and browser management explained in the following sections.
```
Comment (titusfortner) [inline website_and_docs/content/documentation/selenium_manager.en.md:24]:
```suggestion
Selenium Manager is a CLI (command line interface) tool implemented in Rust to allow cross-platform execution and compiled for Windows, Linux, and macOS. The Selenium Manager binaries are shipped with each Selenium release. This way, each Selenium binding language invokes the correct Selenium Manager executable to carry out the automated driver and browser management explained in the following sections.
```

File: website_and_docs/content/documentation/selenium_manager.en.md [hunk 6]
```diff
@@ -3,42 +3,52 @@ title: "Selenium Manager (Beta)"
...
 4. Driver cache. Uncompressed driver binaries are stored in a local cache folder (`~/.cache/selenium`). The next time the same driver is required, it will be used from there if the driver is already in the cache.
 
 ## Automated browser management
-As of Selenium 4.11.0, Selenium Manager also implements *automated browser management*. With this feature, Selenium Manager allows us to discover, download, and cache the different browser releases, making them seamlessly available for Selenium. The browser automatically managed by Selenium Manager are:
+**TL;DR**: Selenium Manager **automatically discovers, downloads, and caches the browsers** driven with Selenium (Chrome, Firefox, and Edge) when these browsers are not installed in the local system.
+
+As of Selenium 4.11.0, Selenium Manager also implements *automated browser management*. With this feature, Selenium Manager allows us to discover, download, and cache the different browser releases, making them seamlessly available for Selenium. Internally, Selenium Manager uses an equivalent management procedure explained in the section before, but this time, for browser releases.
```
Comment (titusfortner) [inline website_and_docs/content/documentation/selenium_manager.en.md:39]:
```suggestion
As of Selenium 4.11.0, Selenium Manager also implements *automated browser management* for Chrome releases 113+ with [Chrome for Testing (CfT)](https://googlechromelabs.github.io/chrome-for-testing/). With this feature, Selenium Manager allows us to discover, download, and cache the browser version requested in the Browser Options class, making them seamlessly available for Selenium. Internally, Selenium Manager uses an equivalent management procedure explained in the section before, but this time, for browser releases.
```

File: website_and_docs/content/documentation/selenium_manager.en.md [hunk 7]
```diff
@@ -3,42 +3,52 @@ title: "Selenium Manager (Beta)"
...
+
+The browser automatically managed by Selenium Manager are:
```
Comment (titusfortner) [inline website_and_docs/content/documentation/selenium_manager.en.md:41]: I think the CfT info belongs in previous paragraph and for now this section should just mention that Firefox and Edge are under development.

File: website_and_docs/content/documentation/selenium_manager.en.md [hunk 8]
```diff
@@ -50,7 +60,9 @@ But there is even more. In addition to fixed browser versions (e.g., `113`, `114
 When these labels are specified, Selenium Manager first checks if a given browser is already installed (`beta`, `dev`, etc.), and when it is not detected, the browser is automatically managed.
 
 ## Configuration
-Like any other CLI tool, Selenium Manager is invoked from the shell, and different arguments can be used to specify specific capabilities. You can check the different arguments supported by Selenium Manager by running the following command:
+**TL;DR**: Selenium Manager should work silently and transparently for most users. Nevertheless, there are scenarios (e.g., when a proxy or firewall forbids making network requests) where custom configuration is required.
```
Comment (titusfortner) [inline website_and_docs/content/documentation/selenium_manager.en.md]: The bindings do support Proxies out of the box if users specify the proxy information directly in the Options class (as opposed to using args or environment variables)

File: website_and_docs/content/documentation/selenium_manager.en.md [hunk 9]
```diff
@@ -92,11 +104,13 @@ The following table summarizes all the supported arguments supported by Selenium
 In addition to the configuration keys specified in the table before, there are some special cases, namely:
 
 - Browser version. In addition to `browser-version`, we can use the specific configuration keys to specify custom versions per supported browser. This way, the keys `chrome-version`, `firefox-version`, `edge-version`, etc., are supported. The same applies to environment variables (i.e., `SE_CHROME_VERSION`, `SE_FIREFOX_VERSION`, `SE_EDGE_VERSION`, etc.).
-- Driver version. Following the same pattern, we can use `chromedriver-version`, `geckodriver-version`,  `msedgedriver-version`,  etc. (in the configuration file), and `SE_CHROMEDRIVER_VERSION`, `SE_GECKODRIVER_VERSION`, `SE_MSEDGEDRIVER_FIREFOX`,  etc. (as environment variables).
+- Driver version. Following the same pattern, we can use `chromedriver-version`, `geckodriver-version`,  `msedgedriver-version`,  etc. (in the configuration file), and `SE_CHROMEDRIVER_VERSION`, `SE_GECKODRIVER_VERSION`, `SE_MSEDGEDRIVER_VERSION`,  etc. (as environment variables).
```
Comment (titusfortner) [inline website_and_docs/content/documentation/selenium_manager.en.md:107]: I kind of think we should merge browser version and driver version before we get out of beta, but that's a discussion for another time. 😄

File: website_and_docs/content/documentation/selenium_manager.en.md [hunk 10]
```diff
@@ -92,11 +104,13 @@ The following table summarizes all the supported arguments supported by Selenium
...
 - Browser path. Following the same pattern, we can use `chrome-path`, `firefox-path`,  `edge-path`,  etc. (in the configuration file), and `SE_CHROME_PATH`, `SE_FIREFOX_PATH`, `SE_EDGE_PATH`,  etc. (as environment variables).
```
Comment (titusfortner) [inline website_and_docs/content/documentation/selenium_manager.en.md]: I think we want to point out that this is also available inside the bindings from the Options class with the appropriate binary method.

File: website_and_docs/content/documentation/selenium_manager.en.md [hunk 11]
```diff
@@ -118,7 +132,7 @@ Selenium Manager includes two additional arguments two handle the cache, namely:
 Selenium Manager follows the same versioning schema as Selenium. Nevertheless, we use the major version 0 for Selenium Manager releases because it is still in beta. For example, the Selenium Manager binaries shipped with Selenium 4.12.0 corresponds to version 0.4.12.
 
 ## Getting Selenium Manager
-For most users, Selenium Manager should work silently and transparently. But if you want to *play* with Selenium Manager or use it for your use case, you can get the Selenium Manager binaries in different ways:
+For most users, direct interaction with Selenium Manager is not required since the Selenium bindings use it internally. Nevertheless, if you want to *play* with Selenium Manager or use it for your use case involving driver or browser management, you can get the Selenium Manager binaries in different ways:
 
 - From the cache. As of Selenium 4.12.0, the Selenium Manager binaries are extracted from each binding distribution and copied to the cache folder. For instance, the Selenium Manager binary shipped with Selenium 4.12.0 will be stored in the folder `~/.cache/selenium/manager/0.4.12`).
```
Comment (titusfortner) [inline website_and_docs/content/documentation/selenium_manager.en.md]: This isn't going to be supported in 4.12. Too much more to add. Let's say that it's under development.

File: website_and_docs/content/documentation/selenium_manager.en.md [hunk 12]
```diff
@@ -162,9 +176,9 @@ INFO    Browser path: C:\Users\boni\.cache\selenium\chrome\win64\116.0.5845.82\c
 ```
 
 ## Selenium Grid
```
Comment (titusfortner) [inline website_and_docs/content/documentation/selenium_manager.en.md:178]: So, grid is weird right now because you can't use the dynamic aspect of setting the browser version in options. Right now, you need to create a stereotype with the supported versions so that the options will match the node. I think we should change this, but that's still under discussion. 😄 Regardless I think it's a caveat we should spell out.

**Review by titusfortner — CHANGES_REQUESTED** [review]
Overall this is a great explanation. I made some specific comments about things I think should be added/changed.

File: website_and_docs/content/documentation/selenium_manager.en.md (hunk 3 above)
Comment (bonigarcia) [reply website_and_docs/content/documentation/selenium_manager.en.md]: I prefer to avoid this sentence being added in the TLDR. IMO it's evident that things become more stable with more recent releases.

File: website_and_docs/content/documentation/selenium_manager.en.md (hunk 2 above)
Comment (bonigarcia) [reply website_and_docs/content/documentation/selenium_manager.en.md]: Will change.

File: website_and_docs/content/documentation/selenium_manager.en.md (hunk 6 above)
Comment (bonigarcia) [reply website_and_docs/content/documentation/selenium_manager.en.md:39]: This info is explained later, so I would not change this paragraph.

File: website_and_docs/content/documentation/selenium_manager.en.md (hunk 9 above)
Comment (bonigarcia) [reply website_and_docs/content/documentation/selenium_manager.en.md:107]: Browser versions and driver versions are different things. Maybe similar in the case of Chrome and Edge since they follow the same versioning schema for browser and driver, but not Firefox (e.g., Firefox 116 needs geckodriver 0.32.0).

File: website_and_docs/content/documentation/selenium_manager.en.md (hunk 10 above)
Comment (bonigarcia) [reply website_and_docs/content/documentation/selenium_manager.en.md]: I will add a reference to that. I think is this link, right?

https://www.selenium.dev/documentation/webdriver/browsers/edge/#start-browser-in-a-specified-location

That part is currently incomplete, unfortunately.

File: website_and_docs/content/documentation/selenium_manager.en.md (hunk 11 above)
Comment (bonigarcia) [reply website_and_docs/content/documentation/selenium_manager.en.md]: That's a pity. Will change.

File: website_and_docs/content/documentation/selenium_manager.en.md (hunk 12 above)
Comment (bonigarcia) [reply website_and_docs/content/documentation/selenium_manager.en.md:178]: I don't know more about what's going on with Selenium Grid than I explained in this doc. Please extend/change it if required.

File: website_and_docs/content/documentation/selenium_manager.en.md (hunk 3 above)
Comment (titusfortner) [reply website_and_docs/content/documentation/selenium_manager.en.md]: Can we remove the reference to 4.6, then? We actually don't want people to use 4.6, so unless we're doing a version history recap I don't think mentioning it helps.

File: website_and_docs/content/documentation/selenium_manager.en.md (hunk 6 above)
Comment (titusfortner) [reply website_and_docs/content/documentation/selenium_manager.en.md:39]: I think this paragraph overstates what has been implemented, and the clarification in the next paragraph ends up sounding more contradictory rather than explanatory.

I think we may be getting caught up in versioning too much? Maybe we write the whole document specifying that it applies to Selenium 4.12 and avoiding discussing when which feature was implemented (unless we want to add a separate section). Then we just note which features are still in progress.

File: website_and_docs/content/documentation/selenium_manager.en.md (hunk 11 above)
Comment (titusfortner) [reply website_and_docs/content/documentation/selenium_manager.en.md]: I wanted to release 4.12 2 days ago and .NET just kicked my butt. :)

File: website_and_docs/content/documentation/selenium_manager.en.md (hunk 10 above)
Comment (titusfortner) [reply website_and_docs/content/documentation/selenium_manager.en.md]: yeah, there's a different link for Chrome, Edge and Firefox (doesn't apply to IE or Safari)

Comment (titusfortner) [reply website_and_docs/content/documentation/selenium_manager.en.md]: and yeah, would be good to get those examples documented.

File: website_and_docs/content/documentation/selenium_manager.en.md (hunk 12 above)
Comment (titusfortner) [reply website_and_docs/content/documentation/selenium_manager.en.md:178]: What I know is explained in this issue - https://github.com/SeleniumHQ/selenium/issues/12530
My lack of grid experience means I don't actually know how to make it work properly right now, either, is part of the problem. ☹️

File: website_and_docs/content/documentation/selenium_manager.en.md (hunk 8 above)
Comment (bonigarcia) [reply website_and_docs/content/documentation/selenium_manager.en.md]: Changed that TLDR.

File: website_and_docs/content/documentation/selenium_manager.en.md (hunk 3 above)
Comment (bonigarcia) [reply website_and_docs/content/documentation/selenium_manager.en.md]: Done.

File: website_and_docs/content/documentation/selenium_manager.en.md (hunk 2 above)
Comment (bonigarcia) [reply website_and_docs/content/documentation/selenium_manager.en.md]: Done.

File: website_and_docs/content/documentation/selenium_manager.en.md (hunk 6 above)
Comment (bonigarcia) [reply website_and_docs/content/documentation/selenium_manager.en.md:39]: I don't know, perhaps. But I would leave it like that for now (since we are in 4.11.0 and heading to 4.12.0) and review this in a later editing round.

File: website_and_docs/content/documentation/selenium_manager.en.md (hunk 10 above)
Comment (bonigarcia) [reply website_and_docs/content/documentation/selenium_manager.en.md]: I see. I will use Chrome's then.

Comment (bonigarcia) [reply website_and_docs/content/documentation/selenium_manager.en.md]: I added Chrome, Firefox, and Edge.

File: website_and_docs/content/documentation/selenium_manager.en.md (hunk 11 above)
Comment (bonigarcia) [reply website_and_docs/content/documentation/selenium_manager.en.md]: Changed.

**Comment by bonigarcia** [comment]
@titusfortner I have just committed a new path to this PR. I think everything (or most of it) should be fixed now.

**Review by titusfortner — APPROVED** [review]
